<!-- -->
<launch>

	<arg name="detection_range" default="1.0" /> <!-- meter -->


	<!-- rosrun lane_follower distance_keep _detection_range:=0 -->
	<node pkg="lane_follower" type="distance_keep" name="distance_keep" output="screen">
		<param name="detection_range" value="$(arg detection_range)" />
	</node>

</launch>
